config/networks.json

```diff
@@ -373,6 +373,23 @@
     "safeAddress": "0xfC78B018B4daD77351095f00D92934A9A851DA34",
     "gasZipChainId": 29
   },
+  "opbnb": {
+    "name": "opBNB",
+    "chainId": 204,
+    "nativeAddress": "0x0000000000000000000000000000000000000000",
+    "nativeCurrency": "BNB",
+    "wrappedNativeAddress": "0x4200000000000000000000000000000000000006",
+    "status": "active",
+    "type": "mainnet",
+    "rpcUrl": "https://opbnb-mainnet-rpc.bnbchain.org",
+    "verificationType": "etherscan",
+    "explorerUrl": "https://opbnb.bscscan.com",
+    "explorerApiUrl": "https://api-opbnb.bscscan.com/api",
+    "multicallAddress": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "safeApiUrl": "https://safe-transaction-opbnb-mainnet.bnbchain.org/api",
+    "safeAddress": "0xaAD94196680Edb94B421bb5B1E1A0B27db686C72",
+    "gasZipChainId": 59
```

Comment:
this Id is wrong (should be 58 according to https://dev.gas.zip/gas/chain-support/outbound)